packages/services/storage/src/schema-change-model.ts

```diff
@@ -197,15 +197,19 @@ export function implement<Model = never>() {
   };
 }
 
+function replaceQuotes(value: string): string {
+  return value.replaceAll("'", '&apos;').replaceAll('"', '&quot;');
```

Comment:
Good catch. Do we have more things that need to be escaped? 🤔 
@kamilkisiela @n1ru4l @TuvalSimha